flexget/plugins/clients/transmission.py

```diff
@@ -631,7 +631,10 @@ def on_task_output(self, task, config):
                 elif config['action'] == 'resume':
                     self.client.start_torrent([torrent_info.id])
                     logger.info('resumed {} in transmission', torrent_info.name)
-
+                elif config['action'] == 'start now':
+                    self.client.start_torrent([torrent_info.id], bypass_queue=True)
```

Comment:
you should use `partial` object here to avoid code duplication

Reply:
I am unsure what you mean, could you please elaborate?

Reply:
He means partial application using `functools.partial` (https://docs.python.org/3/library/functools.html)

Reply:
Yes, I got that. But I am unaware how it would prevent code duplication since this is just a one liner.
I'll happily change it but I don't know what to.

Reply:
removed some stuff for brevity:
```python
start_torrent = partial(self.client.start_torrent, [torrent_info.id])
...
elif config['action'] == 'resume':
    start_torrent()
elif config['action'] == 'start now':
    start_torrent(bypass_queue=True)
```

Reply:
Wouldn't it be more intuitive to have `bypass_queue` as an optional parameter and pass it with resume?
```diff
                 elif config['action'] == 'resume':
-                    self.client.start_torrent([torrent_info.id])
+                   self.client.start_torrent([torrent_info.id],  bypass_queue=config.get('bypass_queue', False))
                     logger.info('resumed {} in transmission', torrent_info.name)
-                elif config['action'] == 'start now':
-                    self.client.start_torrent([torrent_info.id], bypass_queue=True)
-                    logger.info('started {} in transmission', torrent_info.name)
```

If adding yet another parameter sounds bad, could possibly use the already existing `queue_position` to act as bypass when set to `-1`